src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/Options/GoogleCalendarSettings.cs

```diff
@@ -9,5 +9,15 @@ public class GoogleCalendarSettings
 
         [Required]
         public CalendarEvent Events { get; set; }
+
+        [Required]
+#pragma warning disable CA1056 // Uri properties should not be strings
+        public string CalendarsUrl { get; set; }
```

Comment:
please just rename it to CalendarsEndpoint or sth similar (without Url in the name) to avoid warning